packages/react-components/react-tabs/library/src/components/InteractiveTab/InteractiveTab.types.ts

```diff
@@ -0,0 +1,34 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import type { TabSlots, TabInternalProps, TabInternalState } from '../Tab/Tab.types';
+
+export type InteractiveTabSlots = {
+  /**
+   * Root of the component.
+   */
+  root: Slot<'div'>;
+
+  /**
+   * button
```

Comment:
Consider expanding this comment to be more meaningful.

Reply:
Yes, that was a placeholder for a comment. I will tidy it up before submitting it for review.